Adds k8s_raw tests

chouseknecht opened the pull request:
##### SUMMARY
- Adds additional tasks to `openshift_raw` role
- Adds new role for `k8s_raw`
- Sets openshift image to 3.7.1, to fall within Kubernetes Python client supported range. See [the compatibility matrix](https://github.com/kubernetes-client/python/blob/master/README.md#compatibility).

**NOTE** 
[openshift-restclient-python PR #153](https://github.com/openshift/openshift-restclient-python/pull/153) needs to land before this will work.

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
test/integration/targets/k8s_raw

##### ANSIBLE VERSION
```
ansible 2.6.0 (feature/k8s_testing cbf64a291a) last updated 2018/02/15 02:41:57 (GMT -400)
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/Users/chouseknecht/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/chouseknecht/projects/ansible/lib/ansible
  executable location = /Users/chouseknecht/projects/ansible/bin/ansible
  python version = 2.7.14 (default, Nov 14 2017, 23:24:24) [GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.38)]
```

Commit:
Adds k8s_raw, openshift_raw tests

mattclay reviewed (commented):
These files appear unused:

- KubeObjHelper.log
- loggit.txt
- playbook.yml

Were they accidentally added to the PR?

mattclay commented:
CI failure in integration tests: https://app.shippable.com/github/ansible/ansible/runs/54847/66/tests

chouseknecht commented:
- Removed extraneous log files
- Fixed lint issues
- K8s tests are now working with OpenShfit client v0.4.3
- Moved the OpenShift image version back to 3.7.1. The OpenShift Python client doesn't currently support 3.9.
